x/sequencers/types/sequencer.go

```diff
@@ -7,14 +7,14 @@ import (
 )
 
 /* ---------------------------------- UTILS --------------------------------- */
-func NewSequencer(operator sdk.ValAddress, pubKey cryptotypes.PubKey, power uint64) (stakingtypes.Validator, error) {
+func NewSequencer(operator sdk.ValAddress, pubKey cryptotypes.PubKey, power int64) (stakingtypes.Validator, error) {
 	val, err := stakingtypes.NewValidator(operator, pubKey, stakingtypes.Description{})
 	if err != nil {
 		return stakingtypes.Validator{}, err
 	}
 	if power > 0 {
```

Comment:
We need to validate power == 1 otherwise will clash with nextvalidatorHash assumption on dymint (we encode 1 in multiple places)

Reply:
not sure we need to validate. we use what we get from dymint
it validated by the cosmos-sdk as part of the `InitChain` flow